src/core/main.cpp

```diff
@@ -252,6 +252,9 @@ int main( int argc, char * * argv )
 		return EXIT_FAILURE;
 	}	
 #endif
+#if defined( LMMS_BUILD_LINUX ) && QT_VERSION < 0x050000
```

Comment:
I believe we should set this for all Qt versions with a `TODO` or `FIXME` comment that is currently ineffective for Qt5.

Reply:
Done via f693bf6.